src/sentry/digests/__init__.py

```diff
@@ -23,13 +22,20 @@
 backend.expose(locals())
 
 
-class Record(namedtuple("Record", "key value timestamp")):
+class Record(NamedTuple):
+    key: str
+    value: Any  # TODO: I think this is `Notification` ?
```

Comment:
I tried fixing this as well -- but there are too many callers which stuff other things in here -- or build `Notification` objects with incorrect data

will follow up and fix that later!